functions.php

```diff
@@ -70,105 +108,106 @@ function createMTGCard(array $result) {
     $flavorDesignerText = $result['designerFlavourText'];
     $power = $result['power'];
     $toughness = $result['toughness'];
-
-    echo "<div class='card-back-$color'>";
-    echo "<div class='card-top-container'>";
-    echo "<div class='card-title-container'>";
-    echo "<p>$cardTitle</p>";
-    echo "</div>";
-    echo "<div class='mana-cost-container'>";
-    echo "<div class='mana-cost-display-container'>";
-
+    $outputString = '';
```

Comment:
since there is no foreach loop you don't need this initial declaration. 
Just start off the block with $outputString = "..." rather than $outputString .= "..."